src/features/header/header.module.scss

```diff
@@ -0,0 +1,45 @@
+.header {
+  height: 75px;
+  padding: 0 3rem !important;
```

Comment:
why do u use important?

Reply:
I used it in order to override Bootstrap CSS styles.
Instead of using "!important" - I will remove className "navbar" in <nav>.